### ISSUE TITLE
Error: Catchall - Exception:(-2 Name or Service Not Known')

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#88]
```
What steps will reproduce the problem?
1. On Recordings screen; Click on a Recording to watch it
2. Instead of the detail coming up the error pops up 
3. Clear the error and then the recording plays OK

What is the expected output? What do you see instead?
A recording details screen - an error box pops up with Error: Catchall / 
Exception:(-2 Name or Service Not Known') / Function: <function uninti at 
0xb2......>

What version of the product are you using? On what operating system?
Mythbox 1776 on XBMC on Acer Aspire Revo - apt-get updated
Mythdora 10.22 - yum updated

Please provide any additional information below.
The main recording screen also is not downloading any more cover art for 
the recordings; it seemed to stop after doing 20-40 or so just fine.
```

Original issue reported on code.google.com by `lairdr...@gmail.com` on 17 Feb 2010 at 12:23

**Comment by GoogleCodeExporter** [comment]
```
I'm seeing the same thing, but I'm getting "Exception:(111, 'Connection 
Refused')".
Given that it is a different exception, it might be a different issue.
```

Original comment by `wes...@gmail.com` on 23 Feb 2010 at 2:29

**Comment by GoogleCodeExporter** [comment]
```
I also get that same error but only after the first error I mentioned above 
happens 
and then I think I get what you said above each time.  Love the software 
however and 
XBMC - very nice combo. L
```

Original comment by `lairdr...@gmail.com` on 24 Feb 2010 at 3:03

**Comment by GoogleCodeExporter** [comment]
```
I've been able to hack in a fix, it's pretty dirty right now but I'm going to 
try and
clean up it (IE fix it correctly) and submit a patch.

The problem seems to be around "resources/src/mythbox/mythtv/conn.py" line 173 
(svn 
1798), the slaveBackend variable isn't populated correctly.

conn.py:171: if slaveBackend == None:
conn.py:172:    slaveBackend = self.host
conn.py:173: s.connect((slaveBackend, self.port))
```

Original comment by `wes...@gmail.com` on 24 Feb 2010 at 4:22

**Comment by GoogleCodeExporter** [comment]
```
Think I ran into this problem when I had a slave backend with just a tuner. 
Will post 
a test build for you in a few hours.
```

Original comment by `semir.pa...@gmail.com` on 1 Mar 2010 at 11:31

**Comment by GoogleCodeExporter** [comment]
```
Let me know if this build fixes the problem.
```

Original comment by `semir.pa...@gmail.com` on 2 Mar 2010 at 3:22

Attachments:
- [mythbox-svn-1835.tar.gz](https://storage.googleapis.com/google-code-attachments/mythbox/issue-88/comment-5/mythbox-svn-1835.tar.gz)

**Comment by GoogleCodeExporter** [comment]
```
Nope it did not; same problem but it did seem to get more artwork for some of 
the 
recordings that weren't near the top (again the old build stopped getting art 
after 
a 'page down' or two).  I don't think it ever gave me WesW02's error only the 
oine I 
posted however. The way I installed it was to simply overwrite the old Mythbox 
directory; should I have done something different?
```

Original comment by `lairdr...@gmail.com` on 3 Mar 2010 at 3:24

**Comment by GoogleCodeExporter** [comment]
```
It would be safer to rename your orignal MythBox dir to something else and 
start out 
with a fresh install. Also, rename ~/.xbmc/userdata/script_data/MythBox so that 
old 
cached files are not used. If you still run into the same problem, attach your 
xbmc.log.
```

Original comment by `semir.pa...@gmail.com` on 4 Mar 2010 at 12:23

**Comment by GoogleCodeExporter** [comment]
```
Sorry for the lag; did as you suggested and it still comes back with the same 
error 
as I mentioned above.  I'll attach my xbmc.log file as well and I did a fresh 
boot 
and went stright into Mythbox and tried to pull up the detailed screen of one 
recording that has a picture and another that doesnt have a picture next to it. 
 I 
then went into Watch LV and tried to watch CNN... also doesn't work but it does 
work 
from my MythFrontEnd.  THanks, Laird
```

Original comment by `lairdr...@gmail.com` on 27 Apr 2010 at 2:44

Attachments:
- [xbmc.log](https://storage.googleapis.com/google-code-attachments/mythbox/issue-88/comment-8/xbmc.log)

**Comment by GoogleCodeExporter** [comment]
```
What are the hostname and ip address of your frontend and backend? Can you ping 
your 
backend's hostname (not the ip address) from the frontend?
```

Original comment by `semir.pa...@gmail.com` on 27 Apr 2010 at 3:50

**Comment by GoogleCodeExporter** [comment]
```
MythTv server is 192.168.0.100, gateway is .1 XBMC is .32 but is DHCP'ed. It 
couldn't ping hostnames as I only use IP's; anyway I edited the hosts file on 
XBMC 
and the Gateway and now frontend XBMC can ping the Mythdora box using its 
hostname.  
The error however remains as described above.  Either way, thanks for the great 
product and your prompt replies! Laird
```

Original comment by `lairdr...@gmail.com` on 27 Apr 2010 at 10:55

**Comment by GoogleCodeExporter** [comment]
```
Your issue seems to be all the myth urls stored in your database contain 
127.0.0.1 as 
the ip address (this is you loopback address that is not reachable from other 
computers on your network).  You need to change this to the real ipaddress of 
the 
backend. If you run mythtv-setup on the backend and replace 127.0.0.1 on all 
the 
screens with the real ip address, you should not have any problems.
```

Original comment by `semir.pa...@gmail.com` on 9 May 2010 at 4:01

**Comment by GoogleCodeExporter** [comment]
```
I updated things as you suggested but also upgraded to latest mythdora and now 
the
protocol is too new for mythbox.  i didn't know what you meant when you said 
check
for latest release from hg in your post to issue 92 or 93.. who is hg?  Thanks 
again
for all your work on this I hope to get 'er working again. Laird
```

Original comment by `lairdr...@gmail.com` on 14 May 2010 at 1:39

**Comment by GoogleCodeExporter** [comment]
```
sudo apt-get install mercurial
cd ~/.xbmc/scripts
hg clone http://mythbox.googlecode.com/hg mythbox-hg

```

Original comment by `semir.pa...@gmail.com` on 27 May 2010 at 3:31

**Comment by GoogleCodeExporter** [comment]
```
I am not sure if I should be commenting here or starting a new issue but here 
goes:

What steps will reproduce the problem?
1.Launch MythBox
2.Launch Watch TV
3.Select any channel

What is the expected output? What do you see instead?
Expect to see live TV but get the same error as above.

What version of the product are you using? On what operating system?
Backend: Mythbuntu 10.04 MythTV 0.23
Frontend: Mythbuntu 10.04 Mythbox RC1

I have also created a new issue #98 but I get this problem on another frontend 
machine.
```

Original comment by `a...@ncs.co.nz` on 19 Jun 2010 at 5:43

Attachments:
- [mythbox.log](https://storage.googleapis.com/google-code-attachments/mythbox/issue-88/comment-14/mythbox.log)
- [xbmc.log](https://storage.googleapis.com/google-code-attachments/mythbox/issue-88/comment-14/xbmc.log)

**Comment by GoogleCodeExporter** [comment]
```
Please check fix for issue 98. I believe this is the same issue.
```

Original comment by `semir.pa...@gmail.com` on 24 Jul 2010 at 8:50

**Comment by GoogleCodeExporter** [comment]
```
I checked it - not sure how to integrate the diffs or new code you wrote; I am 
now having trouble with this same error on RC1 but can watch the media from the 
homepage but nothing else from the recordings page. New recordings disappear 
once transcoded but that might be a mythbackend problem...
```

Original comment by `lairdr...@gmail.com` on 27 Jul 2010 at 12:11

**Comment by GoogleCodeExporter** [comment]
```
Get your backend fixed and try the latest trunk code.  Re-open if you have 
reason to believe the problem is with MythBox.
```

Original comment by `semir.pa...@gmail.com` on 8 Aug 2010 at 10:33
- Changed state: **Invalid**
